common/buildings/wc_temple.txt

```diff
@@ -585,11 +584,11 @@ temple = {
 			allow_magic_buildings_trigger = yes
 		}
 
-		gold_cost = 200
-		build_time = 400
+		gold_cost = 210
+		build_time = 365
 
-		casters_offensive = .04
-		casters_morale = .01
+		light_infantry_offensive = .04
+		light_infantry_morale = .01
```

Comment:
because modifiers don't work like this, reeeee

Reply:
Actual? The pain will never end

Reply:
>Actual? 

No, it's not the problem
For 4% buff, use 0.04 instead of .04, otherwise it won't work